src/components/CloseIconComponent.js

```diff
@@ -0,0 +1,12 @@
+import { button, span } from 'react-hyperscript-helpers';
+
+export default function CloseIconComponent(props) {
+  const { closeFn } = props;
+  return button({
+    type: 'button',
+    className: 'modal-close-btn close',
+    onClick: closeFn
+  }, [
+    span({ className: ' glyphicon glyphicon-remove default-color'})
```

Comment:
super minor nit. Prefixing the classname with a space is forward-thinking. We should likely suffix it the same way so it can be combined with other classnames on either side of an assignment. Although, I can envision a class name joiner utility that handles this, so feel free to ignore this if you wish.